chsdi/models/vector/__init__.py

```diff
@@ -100,7 +100,7 @@ def __read__(self):
     def srid(self):
         return self.geometry_column().type.srid
 
-    def to_esrijson(self, trans, returnGeometry):
+    def to_esrijson(self, trans, returnGeometry, out_srid):
```

Comment:
do we need this param? it doesn't seem to be used anywhere

Reply:
We already discussed this during code review. Parameter is not used here, but calling function passes three args (https://github.com/geoadmin/mf-chsdi3/blob/mvt_clean/chsdi/views/features.py#L760), if removed, all calling functions should be changed as well. @procrastinatio needed or not? you set the attribute `wkid=self.geometry_column().type.srid_out` when constructing the Feature, was the intention to use `out_srid` here?